chia/wallet/did_wallet/did_wallet.py

```diff
@@ -1352,3 +1356,209 @@ def deserialize_backup_data(backup_data: str) -> DIDInfo:
             metadata,
         )
         return did_info
+
+    async def mint_nfts(
```

Comment:
Why not put this in the NFT wallet, like the bulk minting tool? It's kind of weird to put this in the DID wallet.